### ISSUE TITLE
This example is similar to how WebSocket code looks in browsers using JavaScript.

### ISSUE THREAD

**Issue by neilyoung** [websocket-client/websocket-client#620]
Hmm. Really? I'm not aware of being forced to run a "run_forever" loop with JS websockets. Which in consequence requires me to do that on a different thread...

**Comment by 1148270327** [comment]
so am I，this client can't send msg using callback-function like JS-websocket,

**Comment by engn33r** [comment]
This comment in the README.md stating `This example is similar to how WebSocket code looks in browsers using JavaScript.` is a matter of opinion, and your opinion may differ. If you compare WebSocketApp to the JavaScript WebSockets API [on Mozilla's website](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API), you'll see onclose(), onerror(), onmessage(), and onopen() event handlers are similar in both. I believe this is the point that is trying to be made by this README.md comment. The threading usage in this library is known to be an area that causes many questions (see [this documentation page](https://websocket-client.readthedocs.io/en/latest/threading.html), which is currently just a placeholder at the time of writing). If you have a specific problem with the handling of threads, you can open a new issue.

**Comment by neilyoung** [comment]
The claim to have a similar WS solution as JS is misleading. It might look like so from the event handlers, but in the end this is just the same camouflage as a Wolf appearing like a Sheep. The bummer comes later with the infinite run loop, which leaves no room to handle a WS connection with the same “by the way” attitude as like in JS. But you are right, it is a matter of opinion. So be happy with your’s. There are fortunately alternatives.

**Comment by engn33r** [comment]
Fair enough, point taken. I was reshuffling the documentation today with commit 833a3df and decided that this opinion was rather out of place in the README, so I removed it. While I may later add some comment specifically pertaining to the similarity of the event handlers with the JavaScript WebSocket API (which the prior comparison never called out, and I agree that a more general comparison is inaccurate), I have now removed this comparison from the README. Thank you for your opinion!
